hal/armv7a/cpu.c

```diff
@@ -199,3 +199,14 @@ char *hal_cpuFeatures(char *features, unsigned int len)
 
 	return features;
 }
+
+
+/* cache management */
+
+
+void hal_cleanDCache(ptr_t start, size_t len)
+{
+	(void)start;
+	(void)len;
+	/* TODO */
+}
```

Comment:
Cortex-A has already defined function for data cache cleaning:
https://github.com/phoenix-rtos/phoenix-rtos-kernel/blob/ae0a22325e93d3f9d921a1feffd089e52de1463b/hal/armv7a/armv7a.h#L53-L54

Reply:
Done